pkg/log/zap/zap_test.go

```diff
@@ -255,6 +255,15 @@ var _ = Describe("Zap logger setup", func() {
 					"namespace": name.Namespace,
 				}))
 			})
+
+			It("should not panic with nil obj", func() {
+				pod := &corev1.Pod{}
```

Comment:
Nit to get a typed nilpointer:
```suggestion
				var pod *corev1.Pod
```